app/controllers/concerns/foreman/controller/registration_commands.rb

```diff
@@ -1,5 +1,6 @@
 module Foreman::Controller::RegistrationCommands
   extend ActiveSupport::Concern
+  include ProvisioningTemplatesHelper
```

Comment:
Why is it being explicitly included here? I don't see any change that uses those helpers.

EDIT: I see now, it's for `download_utility` method...